packages/cadl-vs/src/VSExtension.cs

```diff
@@ -17,12 +17,14 @@
 using Task = System.Threading.Tasks.Task;
 using System.Linq;
 
-namespace Microsoft.Cadl.VisualStudio {
+namespace Microsoft.Cadl.VisualStudio
+{
   [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
   [Guid("88b9492f-c019-492c-8aeb-f325a7e4cf23")]
   public sealed class Package : AsyncPackage { }
 
-  public sealed class ContentDefinition {
+  public sealed class ContentDefinition
+  {
```

Comment:
I'm suprised we're not getting four spaces here. Is it detecting indentation?

Reply:
oh we still have indent_size = 2 for `[*]`. As long as we're switching to C# standard style, we should set indent_size to 4 for C#.

Reply:
![image](https://user-images.githubusercontent.com/1031227/165856044-d9a917c2-a2a8-46f1-b453-5e916354bf2c.png)
seems like we are still setting the indent size to 2 for everything